src/frontend/src/components/forms/fields/DateField.tsx

```diff
@@ -11,20 +13,25 @@ export default function DateField({
   controller: UseControllerReturn<FieldValues, any>;
   definition: ApiFormFieldType;
 }) {
+  dayjs.extend(customParseFormat);
```

Comment:
Wouldn't this make sense to move this outside of the component into the global scope of this file, as this function is called a lot of times (on every change of the value)?

Reply:
I am new to the PUI and the typescipt thing, where is exactly  the global scope of this file defined?

Reply:
Just outside the function directly in the file after the imports. Because this line doesn't seem to be dependent on any react stuff.

Reply:
Ah thanks, I thought that it should be moved to some more global file.